tests/test.py

```diff
@@ -4331,6 +4332,29 @@ def testUpperConfigUpperQuery(self):
         self.runTestCase("UPPERCASE_TABLE", NOT_QUOTED, "data", NOT_QUOTED, SHOULD_MATCH)
 
 
+class TestSigTERMHandler(BaseTestCase):
+    def setUp(self):
+        super().setUp()
+
+    def fork_acra(self, popen_kwargs: dict = None, **acra_kwargs: dict):
```

Comment:
do we really need explicit declaration of `setUp` and `fork_acra` if we don't override them?
default [setUp](https://github.com/cossacklabs/acra/blob/master/tests/test_common.py#L394) already assigns process to `self.acra` variable